src/common/GroupAvatar.js

```diff
@@ -57,7 +57,7 @@ export default class GroupAvatar extends PureComponent<Props> {
     return (
       <Touchable onPress={onPress}>
         <View style={[styles.frame, frameSize]}>
-          <Text style={[styles.text, textSize]}>{initialsFromString(names)}</Text>
+          <Text style={[styles.text, textSize]}>{initialsForGroupIcon(names)}</Text>
```

Comment:
Since this was the only occurrence of `initialsFromString`, this commit should probably also remove it.